docs/reference/scripts.md

```diff
@@ -15,4 +15,9 @@ appium driver run xcuitest <script-name>
 |Script Name|Description|
 |------------|-----------|
 |`open-wda`|Opens the WebDriverAgent project in Xcode|
-|`build-wda`|Builds the WebDriverAgent project using the first available iPhone simulator and the latest iOS supported by the current Xcode version|
+|`build-wda`|Builds the WebDriverAgent project using the first available iPhone simulator and the latest iOS supported by the current Xcode version by default. Params `--sdk` and `--name` to customize iOS version and the device - if not specified latest iOS and first available iPhone simulator|
+
+
+```bash
```

Comment:
Please include this example into the table call above

Reply:
done. I hope you like it this time.

If not let me know how you see it 😏